assignments/callbacks.js

```diff
@@ -24,25 +24,51 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length)
 }
+getLength(items,(arr1) => {
+  console.log(arr1)
+})
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[3])
```

Comment:
This works only for the given data set, how could you change this answer so that it was more dynamic and could return the correct answer for this question if the data set was changed or updated?